packages/components/modules/messages/CreateChatRoomList/ChatRoomListItem/index.tsx

```diff
@@ -48,9 +46,14 @@ const ChatRoomListItem: FC<ChatRoomListItemProps> = ({
             commit({
               variables: {
                 input: { profileId: currentProfile.id, participants: [id] },
+                connections: [
+                  // TODO: add filter handling (for now we can default 'unreadMessages' to false)
+                  ConnectionHandler.getConnectionID(currentProfile.id, 'roomsList_chatRooms', {
+                    unreadMessages: false,
+                  }),
+                ],
               },
-              onCompleted: (data) => {
-                setChatRoom({ id: data?.chatRoomCreate?.room?.node?.id })
```

Comment:
Why are we removing this ? should the behavior be that we select the created room so that the user can already send a message there ?

Reply:
Yes, it's being done inside the `useCreateChatRoomMutation` hook.